opensearch_py_ml/dataframe.py

```diff
@@ -424,9 +424,36 @@ def drop(
             axis = pd.DataFrame._get_axis_name(axis)
             axes = {axis: labels}
         elif index is not None or columns is not None:
```

Comment:
Kind of confused here the parent branch is checking that if one of them is not None but inside its checking again
Line 431 and 440
 maybe this could simplified to what @pyek-bot stated about creating a wrapper for convertToList if needed

Reply:
Fixed